libraries/AP_NavEKF2/AP_NavEKF2_Control.cpp

```diff
@@ -160,13 +160,17 @@ void NavEKF2_core::setAidingMode()
     // Determine when to commence aiding for inertial navigation
     // Save the previous status so we can detect when it has changed
     prevIsAiding = isAiding;
-    // Don't allow filter to start position or velocity aiding until the tilt and yaw alignment is complete
-    bool filterIsStable = tiltAlignComplete && yawAlignComplete;
-    // If GPS usage has been prohiited then we use flow aiding provided optical flow data is present
-    bool useFlowAiding = (frontend->_fusionModeGPS == 3) && optFlowDataPresent();
-    // Start aiding if we have a source of aiding data and the filter attitude algnment is complete
-    // Latch to on
-    isAiding = ((readyToUseGPS() || useFlowAiding) && filterIsStable) || isAiding;
+    // perform aiding checks if not aiding
+     if (!isAiding) {
+        // Don't allow filter to start position or velocity aiding until the tilt and yaw alignment is complete
+        // and IMU gyro bias estimates have stabilised
+        bool filterIsStable = tiltAlignComplete && yawAlignComplete && checkGyroCalStatus();
+        // If GPS usage has been prohiited then we use flow aiding provided optical flow data is present
+        bool useFlowAiding = (frontend->_fusionModeGPS == 3) && optFlowDataPresent();
+        // Start aiding if we have a source of aiding data and the filter attitude algnment is complete
+        // Latch to on
+        isAiding = (readyToUseGPS() || useFlowAiding) && filterIsStable;
+    }
```

Comment:
@priseborough Sorry if this comment is a waste of your time, but I prefer to do that than let what might be a bug pass by. Let me start by saying that this isn't exactly related to this PR, but something I noticed while reviewing it.

The change above (besides adding the check to gyro status) is a non-functional change that makes more obvious that _isAiding_ can only be changed from false to true - once it is true the code above doesn't run. My doubts about this come as I read the comments below:
- "check to see if we are starting or stopping aiding"
- "held until aiding is stopped"
- "We have ceased aiding"

I know that the filter can be re-initialized (and isAiding would return to false), but in that case the code below (when _isAiding_ is false) wouldn't run anyway because _prevIsAiding_ is also reset to false.

So, my question is: should the aiding be able to be stopped or is the code below for that case extra (and never run)?